src/components/like/likeButton.test.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
@Ilithor 
## Please review this set of tests

Reply:
Looks good